source/main.coffee

```diff
@@ -34,45 +37,41 @@ uncacheable = new Set [
   "JSXChild", "JSXChildren", "JSXNestedChildren", "JSXMixedChildren"
 ]
 
-module.exports =
-  parse: parse
-  compile: (src, options=defaultOptions) ->
-    filename = options.filename or "unknown"
-
-    # TODO: This makes source maps slightly off in the first line.
-    if filename.endsWith('.coffee') and not
-       /^(#![^\r\n]*(\r\n|\n|\r))?\s*['"]civet/.test src
-      src = "\"civet coffeeCompat\"; #{src}"
-
-    if !options.noCache
-      events = makeCache()
-
-    ast = prune parse(src, {
-      filename
-      events
-    })
-
-    if options.ast
-      return ast
-
-    if options.sourceMap or options.inlineMap
-      sm = SourceMap(src)
-      options.updateSourceMap = sm.updateSourceMap
-      code = gen ast, options
-
-      if options.inlineMap
-        srcMapJSON = sm.json(filename, "")
-        # NOTE: separate comment to prevent this string getting picked up as actual sourceMappingURL in tools
-        return "#{code}\n#{"//#"} sourceMappingURL=data:application/json;base64,#{base64Encode JSON.stringify(srcMapJSON)}\n"
-      else
-        return {
-          code,
-          sourceMap: sm
-        }
-
-    gen ast, options
-  generate: gen
-  util: util
+export compile = (src, options=defaultOptions) ->
+  filename = options.filename or "unknown"
+
+  # TODO: This makes source maps slightly off in the first line.
+  if filename.endsWith('.coffee') and not
+     /^(#![^\r\n]*(\r\n|\n|\r))?\s*['"]civet/.test src
+    src = "\"civet coffeeCompat\"; #{src}"
+
+  if !options.noCache
+    events = makeCache()
+
+  ast = prune parse(src, {
+    filename
+    events
+  })
+
+  if options.ast
+    return ast
+
+  if options.sourceMap or options.inlineMap
+    sm = SourceMap(src)
+    options.updateSourceMap = sm.updateSourceMap
+    code = generate ast, options
+
+    if options.inlineMap
+      srcMapJSON = sm.json(filename, "")
+      # NOTE: separate comment to prevent this string getting picked up as actual sourceMappingURL in tools
+      return "#{code}\n#{"//#"} sourceMappingURL=data:application/json;base64,#{base64Encode JSON.stringify(srcMapJSON)}\n"
+    else
+      return {
+        code,
+        sourceMap: sm
+      }
+
+  generate ast, options
 
```

Comment:
We may also want add a default export object that holds all of the named exports as a convenience to the importer.